src/vellum/workflows/nodes/displayable/search_node/tests/test_node.py

```diff
@@ -81,7 +86,12 @@ class MySearchNode(SearchNode):
                         lhs_variable=StringVellumValueRequest(value="TYPE"),
                         operator="=",
                         rhs_variable=StringVellumValueRequest(value="COMPANY"),
-                    )
+                    ),
+                    VellumValueLogicalConditionRequest(
+                        lhs_variable=StringVellumValueRequest(value="NAME"),
+                        operator="notNull",
+                        rhs_variable=JsonVellumValueRequest(value=None),
```

Comment:
`NullVellumValue`s are not in our sdk client currently. rhs is ignored during serialization of unary operators anyway